google-cloud-bigtable/google-cloud-bigtable.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = ">= 2.7"
 
   gem.add_dependency "concurrent-ruby", "~> 1.0"
-  gem.add_dependency "google-cloud-bigtable-admin-v2", "~> 0.0"
-  gem.add_dependency "google-cloud-bigtable-v2", "~> 0.14"
+  gem.add_dependency "google-cloud-bigtable-admin-v2", "< 2.a"
```

Comment:
I would still keep it explicit and consistent: `">= 0.0", "< 2.a"`

Reply:
Thanks, updated them all!